examples/watchasay/app/schema.json

```diff
@@ -0,0 +1,15 @@
+{
+    "$id": "https://automate.globus.org/skeleton_action_provider.input.schema.json",
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "title": "Skeleton Action Provider Input Schema",
+    "type": "object",
+    "properties": {
+        "input_string": {
```

Comment:
Maybe good to add a description property here more or less to set the example that it is good to document your properties this way.

Reply:
This would be good too -- I didn't even know the description keyword was an accepted property.  I expect whatever we document here will get copied around so adding it will make a difference.

Reply:
I updated both examples' schema's property definitions with the description